RoosterPlanner.Api/Controllers/ShiftController.cs

```diff
@@ -57,6 +57,31 @@ public async Task<ActionResult<List<ShiftViewModel>>> GetShiftsAsync(Guid id)
             }
         }
 
+        [HttpGet("{id}/{userId}/{date}")]
```

Comment:
Als je een Shift Id meekrijgt, is 'date' dan nog relevant (Id is uniek.. dus een specifieke shift)

Reply:
Ik had dat beter moeten aangeven. Het gaat om een projectId